grc/core/FlowGraph.py

```diff
@@ -373,7 +373,7 @@ def import_data(self, data):
         def verify_and_get_port(key, block, dir):
             ports = block.sinks if dir == 'sink' else block.sources
             for port in ports:
-                if key == port.key or key + '0' == port.key:
+                if key == port.name or key + '0' == port.name or key == port.key:
```

Comment:
a bit confused about this. Do ports now have both an ID and a name?

Reply:
I really can't say what's the right way here. This is as close as I got to a working solution, which still falls short for some connections...